nav2_dwb_controller/dwb_controller/include/dwb_controller/dwb_controller.hpp

```diff
@@ -22,10 +22,13 @@
 #include "dwb_core/common_types.hpp"
 #include "nav_2d_msgs/msg/pose2_d_stamped.hpp"
 #include "nav_2d_utils/odom_subscriber.hpp"
+#include "nav2_world_model/world_model_client.hpp"
 
 namespace nav2_dwb_controller
 {
 
+enum class TravelDirection {MovingForward, MovingBackwards, Stopped};
```

Comment:
Can this be expanded and made explicit? ie
```
enum TravelDirection
{
  MovingForward = 1
  Moving...
}

```

Reply:
The intent here was actually to expand the enum out so its not a 1 liner that's easily glossed over (also not sure the enum class is needed, just enum is sufficient?). My general preference is to remove it, I think it's not state information that needs to be stored. Explicitly defining the numbers is just habit, I suppose it doesn't add value